exercises/two-fer/TwoFerTest.m

```diff
@@ -0,0 +1,27 @@
+#import <XCTest/XCTest.h>
+
+#if __has_include("TwoFerExample.h")
+# import "TwoFerExample.h"
+# else
+# import "TwoFer.h"
+#endif
+
+@interface TwoFerTest : XCTestCase
+
+@end
+
+@implementation TwoFerTest
+
+- (void)testNoNameGiven {
```

Comment:
What about “”?

Reply:
What do you think of a non null region for the test. 
https://developer.apple.com/swift/blog/?id=25

Reply:
Hmm, maybe this would be a good chance to show the `nonnull` and `nullable` annotations for parameters if we keep the `nil` for name? See what you think.